amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-examples.adoc

```diff
@@ -1,11 +1,6 @@
 = Amazon S3 Connector 5.7 Examples
```

Comment:
Please add the text "Paste this code..." before the XML. This is in the template for the Example topic.